helm/oms-agent/Chart.yaml

```
@@ -0,0 +1,12 @@
+apiVersion: v1
+appVersion: 0.0.1
+description: A Helm chart for oms-agent
+home: https://github.com/giantswarm/oms-agent-app
+icon: https://raw.githubusercontent.com/microsoft/Docker-Provider/ci_prod/img/azuremonitor-containers.svg
+name: oms-agent
+sources:
+  - https://github.com/microsoft/Docker-Provider/tree/ci_prod
+  - https://docs.microsoft.com/en-us/azure/monitoring/monitoring-container-health
+restrictions:
+  compatibleProviders: ["azure"]
+version: [[ .Version ]]
```

Would be nice if you would keep this line aligned with the actual helm chart version

---

what do you mean?

---

You're probably planning to release a 0.1.0 after merging this PR, the `[[``]]` templates are only used by `architect`, so not required here.
I suggest setting this to `0.1.0`
```suggestion
version: 0.1.0
```

---

Done